src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,53 @@
 'use strict';
 
+const alive = [];
```

Comment:
Make it a static field of Animal
```suggestion
Animal.alive = [];
```